CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+# Contributing
+
+**🎉 First off, thanks for contributing! 🎉**
+
+- [✨ Types of contributions](#types-of-contributions)
+- [📤 Pull Requests](#pull-requests)
+- [🚀 Development Workflow](#development-workflow)
+- [🔀 Release process](#release-process)
+- [🔬 Testing](#testing)
+- [🎨 Code style](#code-style)
+- [📄 Contributor license agreement](#contributor-license-agreement)
+
+## ✨ Types of contributions
+
+We welcome all types of contributions, including bug fixes, feature enhancements,
+bug reports, documentation, graphics, and many others.  You might consider contributing by:
+
+- Report a bug or request a new feature in our [issue tracker](https://github.com/DataONEorg/REPONAME/issues)
+- Fix a bug and contribute the code with a Pull Request
+- Write or edit some documentation
+- Sharing helpful tips or FAQ-type answers to users or future contributors
+- Create screenshots or tutorials of features of MetacatUI
+- Answer questions on DataONE Discussions
+- ...
+
+This is an open source project, and we welcome full
+participation in the project.  Contributions are reviewed and suggestions are
+made to increase value to the community.  We strive to
+incorporate code, documentation, and other useful contributions quickly and
+efficiently while maintaining a high-quality software product.
+
+## 📤 Pull Requests
+We use the pull-request model for contributions. See [GitHub's help on pull-requests](https://help.github.com/articles/about-pull-requests/).
+
+In short:
+
+- add an [issue](https://github.com/DataONEorg/REPONAME/issues) describing your planned changes, or add a comment to an existing issue;
```

Comment:
I assume we want to link to https://github.com/NCEAS/vegbank2/issues

Reply:
yes, fixed locally and I will push shortly.